src/components/drawer/index.tsx

```diff
@@ -244,22 +248,22 @@ class NewDrawer extends PureBaseComponent {
   /** Renders */
 
   // TODO: enable support for rendering more than one left item
-  renderLeftActions = (progress, dragX) => {
-    const {leftItem} = this.getThemeProps();
+  private renderLeftActions = (progress: Animated.Value/* , dragX: Animated.Value */) => {
+    const {leftItem} = this.props;
     const leftItems = leftItem ? [leftItem] : undefined;
-    return this.renderActions(leftItems, progress, dragX);
+    return this.renderActions(leftItems, progress/* , dragX */);
   };
 
-  renderRightActions = (progress, dragX) => {
-    const {rightItems} = this.getThemeProps();
-    return this.renderActions(rightItems, progress, dragX);
+  private renderRightActions = (progress: Animated.Value/* , dragX: Animated.Value */) => {
+    const {rightItems} = this.props;
+    return this.renderActions(rightItems, progress/* , dragX */);
   };
 
-  renderActions(items, progress, dragX) {
+  private renderActions(items: ItemProps[] | undefined, progress: Animated.Value/* , dragX: Animated.Value */) {
     if (items) {
       return (
-        <View animated row style={{transform: [{translateX: this.leftActionX}]}}>
-          {_.map(items, (item, index) => {
+        <View animated row style={{transform: [{translateX: this.leftActionX}]} as any as ViewStyle}>
```

Comment:
Why you need ' as any as ViewStyle' here?

Reply:
TBH, not sure why it's even giving me an error. 
This is a trick I learned from the guys from Kiev. 
First you cast to any and from there you can cast to anything you want. 

Not a fan of this solution. another option is to explicitly `ts-ignore` this line and revisit in the future. 
WDYT?

Reply:
I would ts-ignore it, the casting seems redundant and expensive

Reply:
Done